Dockerfile.testing

```diff
@@ -1,24 +1,21 @@
 # For building gcr.io/forgerock-io/secret-agent-testing:latest
 
-FROM openjdk:11-jdk-slim
+FROM openjdk:11-jre-slim
 ENV CGO_ENABLED=0 GOOS=linux GOARCH=amd64
-CMD ["bash"]
-RUN apt-get update && apt-get install -y curl git-core
-RUN curl -O https://dl.google.com/go/go1.13.10.linux-amd64.tar.gz && \
-    SUM=$(sha256sum go1.13.10.linux-amd64.tar.gz | awk '{print $1}') && \
+RUN apt-get update && apt-get install -y curl git-core make && apt-get clean all
```

Comment:
`DEBIAN_FRONTEND=noninteractive` should be set as ENV (if it's no in upstream)
also `--no-install-recommends --yes`